lib/screens/consumer_bag/consumer_bag_screen.dart

```
@@ -0,0 +1,424 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:vue_app/screens/consumer_bag/consumer_bag_mock_data.dart';
+import 'package:vue_app/util/constants.dart';
+import 'package:vue_app/util/util.dart';
+import 'consumer_bag_mock_data.dart';
+
+class consumer_Bag extends StatefulWidget {
+  @override
+  _consumer_BagState createState() => _consumer_BagState();
+}
+
+class _consumer_BagState extends State<consumer_Bag> {
+  int checkboxItem = 0;
+  double checkboxPrice = 0;
+
+  int totalItem() {
+    int totalNum = 0;
+    for (var g in orderUnderBrand) {
+      for (var i in g.addedItemsFromBrand) {
+        totalNum += i.numberOfItem;
+      }
+    }
+    return totalNum + checkboxItem;
+  }
+
+  double totalPrice() {
+    double totalPrice = 0;
+    for (var g in orderUnderBrand) {
+      for (var i in g.addedItemsFromBrand) {
+        totalPrice += i.priceOfItem * i.numberOfItem;
+      }
+    }
+    return totalPrice + checkboxPrice;
+  }
+
+//  int totalItem() {
+//
+//  }
+
+  @override
+  Widget build(BuildContext context) {
```

The build function here is too huge. Make into smaller chunks of functions to improve readability and maintability

---

This one i need a little time cause when i put the function under the mock data it doesnt change and i cannot bring in the function into the state as it has error in it